apps/airview-demo/src/views/document-view/external-content.js

```diff
@@ -8,7 +8,7 @@ export function ExternalContent({ metadata }) {
 
   useEffect(() => {
     async function getData() {
-      const url = `/api/external-content/${metadata.external_repo}/${metadata.external_owner}?path=${metadata.external_path}`;
+      const url = `/api/cms/external-content/${metadata.external_repo}/${metadata.external_owner}?path=${metadata.external_path}`;
```

Comment:
Good catch!

Can you update to use the variable from the [config](https://github.com/AirWalk-Digital/airview/blob/0d1e440fe6f2d6e887dac249b5cfd80472cce610/apps/airview-demo/src/config.js#L5), as per [this](https://github.com/AirWalk-Digital/airview/blob/0d1e440fe6f2d6e887dac249b5cfd80472cce610/apps/airview-demo/src/views/document-view/document-view.js#L26) API call